codecov.yml

```diff
@@ -1,11 +1,21 @@
 codecov:
   require_ci_to_pass: no # We have several CI jobs that upload codecov results,
                          # and sometimes some of them may fail, it's OK.
+
 coverage:
-  status:
-    project:
-      default: # This can be anything, but it needs to exist as the name
-        if_ci_failed: error #success, failure, error, ignore
-        only_pulls: false
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
Do we really need to set anything here?

Reply:
We don't, these are the defaults and the documentation says that the content of codecov.yml do not overwrite the whole set of settings, it only changes the specified ones. So I think we may keep only `require_ci_to_pass: no`.

Fixed.